samples/system-test/logs.test.js

```diff
@@ -16,27 +16,26 @@
 'use strict';
 
 const path = require(`path`);
-const test = require(`ava`);
 const tools = require(`@google-cloud/nodejs-repo-tools`);
 const uuid = require(`uuid`);
-
+const assert = require('assert');
 const cwd = path.join(__dirname, `..`);
 const cmd = `node logs.js`;
 
 const logName = `nodejs-logging-logs-test-${uuid.v4()}`;
 const message = `Hello world!`;
 
-test.before(tools.checkCredentials);
+before(() => tools.checkCredentials());
 
-test.serial(`should write a log entry`, async t => {
+it(`should write a log entry`, async () => {
```

Comment:
My understanding is that mocha runs tests sequentially. It's OK for this library, but in some others, e.g. spanner or dlp, it will add significant time to the samples tests.